reflectable_builder/build.yaml

```diff
@@ -1,7 +1,7 @@
 builders:
   reflectable:
```

Comment:
Consider renaming this to `reflectable_builder` as well, which will allow the shorthand syntax of just the package name to refer to it.